CI/build/conf/cores_config_ci.json

```diff
@@ -49,6 +49,7 @@
             "GENERIC_C031C4UX",
             "GENERIC_C031C6TX",
             "GENERIC_C031F4PX",
+            "GENERIC_C071R8TX",
```

Comment:
Did you miss GENERIC_C071RBTX here as well?

Reply:
Ditto